chainer/links/normalization/batch_normalization.py

```diff
@@ -66,15 +66,18 @@ def __init__(self, size, decay=0.9, eps=2e-5, dtype=numpy.float32,
                  initial_gamma=None, initial_beta=None):
```

Comment:
I found these two arguments (`initial_gamma` and `initial_beta`) are not documented. Please consider adding documentation to explain how those values are used for initialization with this PR, or please raise a new issue. A brief description to let users know the default initializer and also the fact that users can specify the way to initialize both `gamma` and `beta`, would be enough.